app/presenters/pom_task_presenter.rb

```diff
@@ -1,6 +1,62 @@
 # frozen_string_literal: true
 
-PomTaskPresenter = Struct.new :offender_name,
-                              :offender_number,
-                              :action_label,
-                              :long_label, keyword_init: true
+class PomTaskPresenter
+  include Rails.application.routes.url_helpers
+
+  # @offender may be either an AllocatedOffender or MpcOffender, depending on the source of the call.
+  # However, all relevant methods can be found on MpcOffender.
+  delegate :offender_name, :offender_first_name, :offender_number, :prison_id, :type, :parole_review_id, to: :@pom_task
+
+  def initialize(pom_task)
+    @pom_task = pom_task
+  end
+
+  def first_name_with_ownership
+    "#{offender_first_name.capitalize}#{offender_first_name.downcase[-1] == 's' ? '\'' : '\'s'}"
+  end
+
+  def action_label
+    case type
+    when :parole_outcome_date
```

Comment:
I am seeing the repeated checking of `type`. Might be cleaner to have a PomTaskPresenter per `type` and have a factory method that returns the correct type of PomTaskPresenter for that task.

ie. ParoleOutcomeDatePomTask, EarlyAllocationDecisionPomTask, GenericPomTask